lotus/client.go

```diff
@@ -77,58 +79,137 @@ func New(maddr ma.Multiaddr, authToken string) (*API, func(), error) {
 }
 
```

Comment:
Decided to add logging at this level, to have a better understanding on errs in packages. As a boundary between lotus client messages and our code.